app/services/plans/update_service.rb

```diff
@@ -87,10 +88,10 @@ def process_charges(plan, params_charges)
           unless plan.attached_to_subscriptions?
             payload_charge[:group_properties]&.map! { |gp| GroupProperty.new(gp) }
 
-            pay_in_advance = payload_charge.delete(:pay_in_advance)
+            invoiceable = payload_charge.delete(:invoiceable)
             min_amount_cents = payload_charge.delete(:min_amount_cents)
             if License.premium?
-              charge.pay_in_advance = pay_in_advance || false
+              charge.invoiceable = invoiceable unless invoiceable.nil?
```

Comment:
Metrics/BlockNesting: Avoid more than 3 levels of block nesting. (https://rubystyle.guide#three-is-the-number-thou-shalt-count)